### PR TITLE
create new qpileup document

### PR DESCRIPTION
**Pull request by ChristinaXu2017**
# Description

New documentation (readthedocs.io) for qpileup is created due to the updating of Qpileup command line usage and INI options, and the Previous document is archived. This new document is for the latest Adamajava which is not yet released.  

## Type of change

Documentation
 
# How Has This Been Tested?
No testing required

# Are WDL Updates Required?

n/a

# Checklist:
n/a

### CODE REVIEW

**Review by grendeloz — COMMENTED** [review]
What was the thinking behind the reorganisation into different files?

**Comment by ChristinaXu2017** [comment]
> What was the thinking behind the reorganisation into different files?

There were several different files, one for each mode.  Now I integrated them into one file (index.md). Tred to add more detailed examples for each mode and more explanation of HDF files, just feel the one file is too big to read.  I also use qbamfilter docs as an example, it adds query examples into a separate file.

File: docs/qpileup/index.md [hunk 1]
```diff
@@ -1,539 +1,169 @@
...
-x sam-1.110.jar
-x jhdfobj.jar
-x jhdf5obj.jar
-x jhdf5.jar
-x jhdf.jar
-~~~~
 
 ## Usage
 
+Before run qpileup, the hdf-java has to be installed and exported, eg.
```
Comment (holmeso) [inline docs/qpileup/index.md:28]: Before **_you_** run qpileup, the hdf-java **_library_** has to be...

**Review by holmeso — COMMENTED** [review]
minor changes

File: docs/qpileup/index.md (hunk 1 above)
Comment (ChristinaXu2017) [reply docs/qpileup/index.md]: fixed, thanks

**Review by holmeso — APPROVED** [review]
looks good

**Pull request merged**